bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/action/ScriptExecutionImpl.java

```diff
@@ -37,15 +35,10 @@
 public class ScriptExecutionImpl implements ScriptExecution {
 
     private final Scheduler scheduler;
-    private final RuleManager ruleManager;
-    private final RuleRegistry ruleRegistry;
 
     @Activate
-    public ScriptExecutionImpl(@Reference RuleRegistry ruleRegistry, @Reference RuleManager ruleManager,
```

Comment:
That's a leftover from trying to also implement `callScript` here, but that's difficult.

Reply:
I was wondering if it was needed to prevent init issues during startup, but if that is the reason it can be safely removed. :+1: